src/components/home/Search.js

```diff
@@ -26,43 +36,76 @@ const useStyles = makeStyles((theme) => ({
   search: {
     margin: theme.spacing(1),
   },
+  searchOptions: {
+    marginTop: theme.spacing(-10),
+    marginLeft: theme.spacing(1),
+  },
 }));
 
-function Search({ handleSearch, isLoading }) {
+function Search({
+  handleSearch,
+  handleClick,
+  isLoading,
+  range,
+  handleRangeChange,
+}) {
   const { t } = useTranslation();
   const classes = useStyles();
   return (
-    <Paper component="div" className={classes.root}>
-      <InputBase
-        id="search"
-        disabled={isLoading}
-        className={classes.search}
-        placeholder={t('Search collections using name or author')}
-        fullWidth
-        margin="none"
-        InputLabelProps={{
-          shrink: true,
-          ariaLabel: 'search collections',
-        }}
-        variant="filled"
-        onChange={handleSearch}
-      />
-      <Divider className={classes.divider} orientation="vertical" />
-      <IconButton
-        color="primary"
-        className={classes.iconButton}
-        aria-label="directions"
-        type="submit"
-      >
-        <SearchIcon />
-      </IconButton>
-    </Paper>
+    <>
+      <Paper component="div" className={classes.root}>
+        <InputBase
+          id="search"
+          disabled={isLoading}
+          className={classes.search}
+          placeholder={t('Search collections using name or author')}
+          fullWidth
+          margin="none"
+          InputLabelProps={{
+            shrink: true,
+            ariaLabel: 'search collections',
+          }}
+          variant="filled"
+          onChange={handleSearch}
+        />
+        <Divider className={classes.divider} orientation="vertical" />
+        <IconButton
+          color="primary"
+          className={classes.iconButton}
+          aria-label="directions"
+          type="submit"
+          onClick={handleClick}
+        >
+          <SearchIcon />
+        </IconButton>
+      </Paper>
+      <FormControl component="fieldset" className={classes.searchOptions}>
+        <FormLabel component="legend">{t('Search Range')}</FormLabel>
+        <RadioGroup row value={range} onChange={handleRangeChange}>
+          {Object.values(RANGES).map((entry) => (
+            <FormControlLabel
+              value={entry.value}
+              control={<Radio color="primary" />}
+              label={t(entry.title)}
+            />
+          ))}
+          <Tooltip title="Use | or & for union/intersection of multiple keywords if choose 'Tag' or 'All'">
```

Comment:
Add a todo that we will later assume intersection.

Reply:
I also commented out this tooltip to avoid confusion